src/extensions/IExtensionUtilities.ts

```diff
@@ -0,0 +1,9 @@
+import { TemplateCompiler } from '../compilation/templateCompiler';
```

Comment:
Please declare IExensionUtilities in the same file as ITemplateExtension.

It's true that it's usually a best practice to have each type on it's own file, but some times if a type is only a relatively small "helper" type it's best to declare it in the "main" type file.

Reply:
Agreed